spec/features/idv/analytics_spec.rb

```diff
@@ -200,7 +200,7 @@
         flow_path: 'standard', step: 'verify', acuant_sdk_upgrade_ab_test_bucket: :default, skip_hybrid_handoff: nil, analytics_id: 'Doc Auth'
       },
       'IdV: doc auth verify proofing results' => {
-        success: true, errors: {}, flow_path: 'standard', address_edited: false, address_line2_present: false, analytics_id: 'Doc Auth', ssn_is_unique: true, step: 'verify', acuant_sdk_upgrade_ab_test_bucket: :default, skip_hybrid_handoff: nil,
```

Comment:
This can be removed because it is now included in `base_proofing_results` right?

Also does `base_proofing_results` need to be splatted in?

Reply:
It's all confusingly organized. 😢 

This instance of `ssn_is_unique` was coming from us shoehorning it into `form_response.extra` which was getting logged in VerifyInfoConcern. I'm pulling that out.

`base_proofing_results` is separately logged in full on the next line. I added it there in ResultsProofingJob in a previous PR. There doesn't seem to be anything consuming `ssn_is_unique` so we're hoping moving where it's logged won't adversely impact anything.

There is some apparent duplication of parameters between the two, but that's a separate opportunity for improvement...

Reply:
> Also does base_proofing_results need to be splatted in?

It does not need to be splatted in. We log it as a hash under the `proofing_results` key. `ssn_is_unique` was previously logged 1 level higher and we duplicated it in `proofing_results`.